app/src/views/private/components/notifications-drawer.vue

```diff
@@ -39,34 +42,78 @@
 			{{ t('no_notifications_copy') }}
 		</v-info>
 
-		<v-table
-			v-else
-			v-model="selection"
-			v-model:headers="tableHeaders"
-			show-select
-			:loading="loading"
-			:items="notifications"
-			item-key="id"
-			@click:row="onRowClick"
-		></v-table>
+		<div v-else class="content">
+			<v-list v-if="loading" class="notifications">
+				<v-skeleton-loader v-for="i in 10" :key="i" :class="{ dense: totalCount > 15 }" />
+			</v-list>
+
+			<v-list v-else class="notifications">
+				<v-list-item
+					v-for="notification in notifications"
+					:key="notification.id"
+					block
+					:dense="totalCount > 15"
+					:clickable="Boolean(notification.message)"
+					@click="toggleNotification(notification.id)"
+				>
+					<div class="header">
+						<v-checkbox
+							:model-value="selection.includes(notification.id)"
+							@update:model-value="toggleSelected(notification.id)"
+						/>
+						<div v-tooltip="notification.subject" class="title">{{ notification.subject }}</div>
+						<div class="spacer" />
+						<div class="time">{{ notification.time }}</div>
```

Comment:
Thoughts on using `v-text-overflow` component for the title and the `datetime` display for the time? It's working perfectly as it is now, but just wondering should we reuse components more (very pedantic question!)

Reply:
Yes, we should 👍 
I've updated it to use `v-text-overflow`. The `datetime` display doesn't come with a tooltip when there's an overflow, though. So I think we should move the logic from `datetime` into a composable which we can then reuse and pass to `v-text-overflow` here.